caravel/models.py

```diff
@@ -1101,6 +1101,13 @@ def fetch_metadata(self):
                     metric_type='sum',
                     expression="SUM({})".format(quoted)
                 ))
+            if dbcol.sum:
```

Comment:
I've talked to Max, please implement a separate checkbox for AVG metric.
User may not expect to add 2 metrics when setting the `sum` checkbox and will be tricky to explain him if you want to add `avg` metrics - please select sum.